src/core/Microsoft.PowerToys.Settings.UI/Views/PowerPreviewPage.xaml

```diff
@@ -110,7 +110,9 @@
                     HorizontalAlignment="Left"
                     Margin="{StaticResource SmallTopBottomMargin}"
                     RelativePanel.Below="DescriptionPanel">
+                <HyperlinkButton NavigateUri="https://aka.ms/powerToysPowerPreviewSettingImage">
                 <Image x:Uid="FileExplorerPreview_Image" Source="https://aka.ms/powerToysPowerPreviewSettingImage" />
```

Comment:
This is weird. Why is this image source pointing the internet while the rest of them are not?

Reply:
The rest of them are in assets... I know it's off-topic but still...

Reply:
either file a bug against it or you could quickly fix it.

Reply:
I will create an issue but fix it within this PR